QEfficient/transformers/models/modeling_auto.py

```diff
@@ -82,7 +73,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: str, *args, **kwargs):
         )  # Remove model_card_name from kwargs for transformers APIs
 
         model = QEFFAutoModelToTransformersAutoModelMap[cls.__name__].from_pretrained(
-            pretrained_model_name_or_path, *args, **kwargs
+            pretrained_model_name_or_path, num_hidden_layers=1, attn_implementation="eager", *args, **kwargs
```

Comment:
Why this num_hidden_layers=1? I think this came while you were testing it out. Please review before pushing.

Also, while setting attn_implementation="eager", we also need to check if kwargs already has that. Otherwise, it will trigger a bug.

Reply:
We don't support any other `attn_implementation` right?
I will raise an error if any other is provided via `kwargs`
Will remove `num_hidden_layers`